packages/connect-voting-disputable/src/helpers/numbers.ts

```diff
@@ -0,0 +1,27 @@
+import { BigNumber, utils } from 'ethers'
+
+export const PCT_DECIMALS = 16 // 100% = 10^18
+
+export const PCT_BASE = BigNumber.from(`100${'0'.repeat(PCT_DECIMALS)}`)
+
+export const bn = (x: string): BigNumber => BigNumber.from(x)
+
+export const formatBn = (number: string | BigNumber, numberDecimals: string | number, formattedDecimals = 2): string => {
```

Comment:
Wonder if we may wan to have `PCT_DECIMALS` as default value for `numberDecimals`.

Reply:
Answered bellow in https://github.com/aragon/connect/pull/196#discussion_r472235428